src/components/modules/Quiz/Question/TrueOrFalse.tsx

```diff
@@ -1,21 +1,36 @@
-import cx from 'classnames'
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
 import Button from 'components/atoms/Button/Button'
-import React, { useState } from 'react'
-import ResultIndicator from './ResultIndicator'
+import { useCallback, useState } from 'react'
 import { QuestionProps } from '.'
+import { useTranslation } from 'react-i18next'
+import { useField } from 'formik'
 
-interface ITrueOrFalseData {
-  text: string
-  isTrue: boolean
-}
+const TrueOrFalse: React.FC<QuestionProps> = ({ number, question }) => {
+  const text = question.question
+  const { t } = useTranslation()
+
+  const [buttonState, setButtonState] = useState<{ [key: string]: { bg: string } }>({
+    true: { bg: 'primary_base' },
+    false: { bg: 'primary_base' },
```

Comment:
It would be easier to understand if you store a state here instead of a style. Something like `selected`, `not_selected`. It would be a lot more simpler in the handleAnswer function.

You can use something like this object to translate the state to the correct style:
```
const stateToStyle = {
selected: { bg: 'success_base' },
not_selected: { bg: 'primary_base' }
}
```

Reply:
Good call this would make it easier to read